src/NuGet/Microsoft.SymStore.Indexers.nuspec

```diff
@@ -0,0 +1,22 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.SymStore.Indexers</id>
+    <summary>Tooling to pull symstore indexes from various file formats</summary>
+    <description>
+        Contains tools to gather symstore indexing information from ELF, MachO, PE, and PDB files
+    </description>
+    <language>en-US</language>
+    <requireLicenseAcceptance>true</requireLicenseAcceptance>
+    <version>$version$</version>
```

Comment:
What version currently gets produced? I want to make sure that if any nuget packages get published they are getting a pre-release version number.